codersdk/templateversions.go

```diff
@@ -56,6 +56,17 @@ type TemplateVersionParameterOption struct {
 	Icon        string `json:"icon"`
 }
 
+// TemplateVersionVariable represents a managed template variable.
+type TemplateVersionVariable struct {
+	Name         string `json:"name"`
+	Description  string `json:"description"`
+	Type         string `json:"type" enums:"string,number,bool"`
```

Comment:
I didn't see that these enum values are enforced anywhere, should they be? (I posted a suggestion in the migration to use database enum.)

Reply:
Replied above